### PR TITLE
Add Contributor page

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) adds a new page to provide basic information about contributing to our user guide in various ways.

If Magento Commerce, please specify:

- [ ] Commerce only?
- [ ] Commerce with B2B?

## Affected documentation pages

Does not affect any existing  pages

Use staging build 450 for review.

### CODE REVIEW

File: src/getting-started/contribute.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+All User Guide pages display a feedback icon.
+
+![Click the feedback icon]({% link getting-started/assets/feedback-icon.png %})
+
+Simply click this icon to provide a rating for the current page and, optionally, give us some actionable feedback about how we can make it better.
+
+## Log an issue
+
+When you see a problem with one of our pages, please let us know. This could be a broken link, content error, or other item that you can identify for fixing.
```
Comment (hickskenh) [inline src/getting-started/contribute.md:32]: not a fan of please in docs.  consider removing

File: src/getting-started/contribute.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+1. At the right of the page title, click **Log an issue**.
+
+   ![Log an issue]({% link getting-started/assets/log-issue.png %}){: width="600px"}
+
+   This loads a _create issue_ page for our GitHub repository. The URL of the page automatically populates in the summary field.
+
+   {:.bs-callout-info}
+   You will need to log in to your GitHub account, or [create a new account][2], to log an issue.
```
Comment (hickskenh) [inline src/getting-started/contribute.md:41]: awkward punctuation.
```suggestion
   To log an issue, you will need to [create a new account][2] or log in to your existing GitHub account.  
```

File: src/getting-started/contribute.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+1. Enter a summary (or leave the default), write a short description of the issue, and then click **Submit new issue**.
+
+   ![Submit a new issue]({% link getting-started/assets/submit-new-issue.png %}){: width="600px"}
+
+## Submit content changes
+
+All of our content resides on GitHub, a well-known source control platform. Anyone can [create a free GitHub account][2] and make content changes and additions through the GitHub web UI. If you are a developer or know your way around git source control and local build environments, you can choose to [work with our source files that way][3].
+
+For content contributions to be accepted and published, whether large or small, you must sign the [Adobe Contributor License Agreement (CLA)][4]. This is then associated with your GitHub account and you do not need to do it again.
```
Comment (hickskenh) [inline src/getting-started/contribute.md:51]:
```suggestion
For content contributions to be accepted and published, whether large or small, you must sign the [Adobe Contributor License Agreement (CLA)][4]. This is then associated with your GitHub account, and you do not need to do it again.
```

File: src/getting-started/contribute.md (hunk 1 above)
Comment (leslietilling) [reply src/getting-started/contribute.md:32]: I don't like it  in our standard stuff either.  But in this case we are asking them for something. But I would totally remove from our product information or tutorials.

**Comment by leslietilling** [comment]
running tests

**Pull request merged**